website/content/docs/connect/cluster-peering/create-manage-peering.mdx

```diff
@@ -7,24 +7,35 @@ description: >-
 
 # Create and Manage Peering Connections
 
-~> **Cluster peering is currently in technical preview:** Functionality associated with cluster peering is subject to change. You should never use the technical preview release in secure environments or production scenarios. Features in technical preview may have performance issues, scaling issues, and limited support.
+~> **Cluster peering is currently in beta:** Functionality associated with cluster peering is subject to change. You should never use the beta release in secure environments or production scenarios. Features in beta may have performance issues, scaling issues, and limited support.
 
-A peering token enables cluster peering between different datacenters. Once you generate a peering token, you can use it to establish a connection between clusters. Then you can export services and authorize other clusters to call those services.
+A peering token enables cluster peering between different datacenters. Once you generate a peering token, you can use it to establish a connection between clusters. Then you can export services and create intentions so that peered clusters can call those services.
 
-To peer clusters, you must complete the following steps in order:
+## Create a peering connection
+
+Cluster peering is not enabled by default on Consul servers. To peer clusters, you must first configure all Consul servers so that `peering` is `enabled`. For additional information, refer to [Configuration Files](/docs/agent/config/config-files).
+
+Then, complete the following steps in order:
```

Comment:
```suggestion
After enabling peering for all Consul servers, complete the following steps in order:
```
For me, starting the paragraph with "Then" has the effect of making the text feel like a step is missing. You might consider making step 1 "Configure all Consul servers to enable so that `peering` is `enabled`."